### DISCUSSION TITLE
If you're using evcc to charge your car - simple script and configuration to stop battery discharge during EV charge

### DISCUSSION THREAD

**Discussion by markusjnagel** [sonnen-tools/node-red#20] [Announcements]
Just a quick note - if you are using evcc for ev charge control then you can use a simple script and config to stop the battery from discharging while you are charging your car:

Here's what I posted at evcc: https://github.com/evcc-io/evcc/discussions/11685
And here is the link (also linked from the discussion above): https://github.com/sonnen-tools/scripts 

If you don't know evcc - check this out: https://evcc.io/ 
I have started using it primarily for PV surplus charging, but now with Tibber, it also supports "cheap hours" charging - which is a blessing:

![image](https://github.com/sonnen-tools/node-red/assets/33287465/e3c6aa35-5f9d-4419-b50a-b9ec815e0586)
(example from just now - over the last weekend, I could charge the car for <= 26ct)

**Comment by Nitrox912** [comment]
Unfortunately, this great function only works in fast charging mode, and the developers are also reluctant to implement this function in PV mode. Is there a possibility to use this function via Home Assistant, for example? Is there possibly an integration for this?
